### ISSUE TITLE
ASF Pre-release 5.5.2.1 crashed

### ISSUE THREAD

**Issue by Lajci** [chr233/ASFEnhance#205]
After ~10 minutes ASF crashed
ASF: Pre-release 5.5.2.1
Plugin: 2.0.10.0

```
2024-01-25 07:45:02|ArchiSteamFarm-2747|FATAL|ASF|OnUnhandledException() System.MissingMethodException: Method not found: 'Boolean ArchiSteamFarm.Steam.Storage.BotConfig.get_AutoSteamSaleEvent()'.
   at ASFEnhance.ASFEnhance.<>c.<<OnASFInit>b__14_0>d.MoveNext()
   at System.Runtime.CompilerServices.AsyncMethodBuilderCore.Start[TStateMachine](TStateMachine&)
   at ASFEnhance.ASFEnhance.<>c.<OnASFInit>b__14_0(Object _)
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext, ContextCallback, Object)
--- End of stack trace from previous location ---
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext, ContextCallback, Object)
   at System.Threading.TimerQueueTimer.Fire(Boolean )
   at System.Threading.TimerQueue.FireNextTimers()
   at System.Threading.ThreadPoolWorkQueue.Dispatch()
   at System.Threading.PortableThreadPool.WorkerThread.WorkerThreadStart()
2024-01-25 07:45:02|ArchiSteamFarm-2747|ERROR|ASF|Exit() Exiting with nonzero error code!
2024-01-25 07:45:02|ArchiSteamFarm-2747|INFO|Microsoft.Hosting.Lifetime|Application is shutting down...
```

**Comment by chr233** [comment]
use ASFEnhance on pre-release ASF is not recommded

**Comment by chr233** [comment]
it seems EXPLORER command not work , avoid using this command may help

**Comment by chr233** [comment]
fixed in 2.0.11.1
